frontend/components/Home/Dashboard/Dashboard.tsx

```diff
@@ -62,6 +60,40 @@ const Dashboard = () => {
     const [logOpen, setLogOpen] = useState(false);
     const [logModal, setLogModal] = useState(false);
 
+    const [messageDisp, setMessageDisp] = useState(false);
```

Comment:
messageDisp logic can remain in the useEffect hook (see below comment). Although strictly calculating it on the fly with a useMemo hook might be better but it's whatever.